src/chartiq_injections/resizing.js

```diff
@@ -1,14 +1,14 @@
 export const maintainSpanSize = () => {
     CIQ.ChartEngine.prototype.prepend('resizeChart', function () {
-        if (this.chart.xaxis && this.chart.xaxis.length) {
+        if (this.isAutoScale && this.chart.xaxis && this.chart.xaxis.length) {
```

Comment:
@morteza-binary the better name is `AutoSize`, you dont need to add `is` to it.

Reply:
@MahboobehMohammadi It'd be better the boolean variables have `is`, `should` or `has` in their names. it helps developers to know the type of the variable.
It's one of the naming convention which we should follow in the future refactorying.

Reply:
@morteza-binary `auto something` true or false will context the meaning but its a nitpick anyway.

Reply:
@MahboobehMohammadi it'd be better to follow the company naming conventions instead. BTW I don't think it is grammatically wrong.